extension/test/integration/make-payment.handler.spec.js

```diff
@@ -1,97 +1,112 @@
 const { expect } = require('chai')
 
 const ctpClientBuilder = require('../../src/ctp')
-const iTSetUp = require('./integration-test-set-up')
-const constants = require('../../src/config/constants')
 const config = require('../../src/config/config')
+const constants = require('../../src/config/constants')
+const iTSetUp = require('./integration-test-set-up')
+
+describe('::make-payment with multiple store use case::', () => {
+  const [commercetoolsProjectKey] = config.getAllCtpProjectKeys()
+  const [
+    adyenMerchantAccount1,
+    adyenMerchantAccount2,
+  ] = config.getAllAdyenMerchantAccounts()
 
-describe('::makePayment::', () => {
-  const adyenMerchantAccount = config.getAllAdyenMerchantAccounts()[0]
-  const commercetoolsProjectKey = config.getAllCtpProjectKeys()[0]
   let ctpClient
 
   beforeEach(async () => {
     ctpClient = ctpClientBuilder.get(commercetoolsProjectKey)
+    await iTSetUp.cleanupCtpResources(ctpClient)
     await iTSetUp.initServerAndExtension({ ctpClient })
   })
 
   afterEach(async () => {
     await iTSetUp.stopRunningServers()
+    await iTSetUp.cleanupCtpResources(ctpClient)
   })
 
   it(
-    'given a payment ' +
+    'given a single commercetools project and payments ' +
       'when makePayment custom field is set and response from Adyen is Authorised, ' +
-      'then it should set key, "makePaymentResponse" custom field, interface interactions ' +
+      'then it should connect the 2 different adyen mechant accounts and ' +
+      'should set key, "makePaymentResponse" custom field, interface interactions ' +
       'and a successfully authorized transaction',
     async () => {
-      const makePaymentRequestDraft = {
-        amount: {
-          currency: 'EUR',
-          value: 1000,
-        },
-        reference: 'YOUR_REFERENCE',
-        paymentMethod: {
-          type: 'scheme',
-          encryptedCardNumber: 'test_4111111111111111',
-          encryptedExpiryMonth: 'test_03',
-          encryptedExpiryYear: 'test_2030',
-          encryptedSecurityCode: 'test_737',
-        },
-        returnUrl: 'https://your-company.com/',
-      }
-      const paymentDraft = {
-        amountPlanned: {
-          currencyCode: 'EUR',
-          centAmount: 1000,
-        },
-        paymentMethodInfo: {
-          paymentInterface: constants.CTP_ADYEN_INTEGRATION,
+      await makePayment({
```

Comment:
I think this way it's better: https://stackoverflow.com/questions/17569382/parametrized-tests-with-mocha. We already used it in https://github.com/commercetools/commercetools-adyen-integration/blob/master/extension/test/e2e/credit-card.spec.js#L42

Reply:
Imo here it's better to run them in parallel in one test server and one compact case, I make calls parallel: 
https://github.com/commercetools/commercetools-adyen-integration/pull/539/commits/7a7c6039e219ec19f8892f0a001becb0b88aa713